platform/darwin/src/MGLNetworkConfiguration.m

```diff
@@ -127,13 +127,15 @@ - (void)cancelDownloadEventForResponse:(NSURLResponse *)response {
     [self sendEventForURLResponse:response withAction:@"cancel"];
 }
 
-- (void)debugLog:(NSString *)format, ... {
-    MGLLogDebug(format);
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-implementations"
+// TODO: Remove in future release.
```

Comment:
Why are we removing this?

Reply:
To bypass a catch-22; I marked the protocol methods as deprecated, and so the compiler warns (errors) if you add them. But if they're missing then the compiler errors about an incomplete conformance. 🤕